src/demo/pages/BlockchainTransaction/SignTransactionSendPayment.tsx

```diff
@@ -145,14 +145,10 @@ const SignTransactionSendPayment = (props: SignTransactionSendPaymentProps) => {
       props.setBalance(updateBalance);
       props.setActiveStep(2);
       props.setProgressValue(60);
-      let url =
-        "https://local.bloks.io/transaction/" +
-        trx?.transaction_id +
-        "?nodeUrl=";
-
-      url += settings.isProduction()
-        ? settings.config.blockchainUrl
-        : "http://localhost:8888";
+      const url =
```

Comment:
see https://github.com/Tonomy-Foundation/Tonomy-ID/pull/932#pullrequestreview-1919934755

Check if the URL path is the same for Open Block Explorer and Bloks.io. If not, add some logic to make sure it works for all environments.